### PR TITLE
Dm 2046

### PR DESCRIPTION
**Pull request by JoshingYou1**
### JIRA issue link
https://agile6.atlassian.net/browse/DM-2046

## Description - what does this code do?
Addresses a plethora of UX tweaks and bug fixes

## Testing done - how did you test it/steps on how can another person can test it 
1. Open this spreadsheet: https://docs.google.com/spreadsheets/d/163NX9FbvAA4Bzsb8ZfrDP5Eg8R41GAX1/edit#gid=1539332384
2. Make sure the issues marked in green have been resolved.

**NOTES**
1. `H5` tags look like this:
Screen Shot 2020-09-30 at 4 25 27 PM
2. For the `date_created` year, Becca recommended that the max character count should be set at 4, with a minimum value of `1970` and maximum of `9999`(or 4 characters).

## Link to mock-ups/mock ups (image file if you have it) (if applicable)
https://www.figma.com/file/FogT5mBUVl8rE7OgWDQWeJ/VADM?node-id=276%3A0

## Acceptance criteria
- [Bookmark not working (May need to revisit this to confirm it’s still an issue)]
- [PII Verbiage]
- [Diffusion Maturity language ]
- [Make H5s consistent ]

## Definition of done
- [ ] Unit tests written (if applicable)
- [ ] e2e/accessibility tests written (if applicable)
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [X] A link has been provided to the originating JIRA issue
- [X] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: app/assets/stylesheets/dm/pages/_practice.scss [hunk 1]
```diff
@@ -359,10 +359,10 @@
   height: 72px !important;
 }
 
-.practice-editor-summary-textarea {
-  resize: none;
-  height: 142px !important;
-}
+//.practice-editor-summary-textarea {
```
Comment (emyl3) [inline app/assets/stylesheets/dm/pages/_practice.scss:362]: nit: remove

File: app/assets/stylesheets/dm_uswds/theme/_uswds-theme-custom-styles.scss [hunk 2]
```diff
@@ -76,23 +76,24 @@ i.e.
 // outline btns - to use: class="usa-button--outline dm-btn-primary"
 .usa-button--outline {
   @extend .usa-button;
-  border-width: 2px;
+  //border-width: 2px;
```
Comment (emyl3) [inline app/assets/stylesheets/dm_uswds/theme/_uswds-theme-custom-styles.scss:79]: nit: remove commented out styles

File: app/views/practices/_breadcrumbs.html.erb [hunk 3]
```diff
@@ -3,160 +3,24 @@
     <a href="/">Home</a>
     <span class="x0-5-left x0-5-right">›</span>
 
-    <% session[:breadcrumbs].each do |b| %>
-      <span><%= link_to b['display'] || b[:display], b['path'] || b[:path] %></span>
-      <span class="x0-5-left x0-5-right">›</span>
+    <%
```
Comment (emyl3) [inline app/views/practices/_breadcrumbs.html.erb:6]: oo really like what you did with the breadcrumbs in this PR 🍞 👍

File: app/controllers/application_controller.rb [hunk 4]
```diff
@@ -45,21 +45,166 @@ def setup_breadcrumb_navigation
...
+      elsif partner_breadcrumb.present? && partner_breadcrumb.count > 1
+        remove_breadcrumb(partner_breadcrumb)
+        add_partner_breadcrumb
       end
     end
+
+    ### PRACTICE EDITOR BREADCRUMBS ###
+    # Instructions breadcrumbs
+    if params[:action] == 'instructions' && params[:controller] == 'practices'
+      practice = get_practice_by_practice_id
```
Comment (emyl3) [inline app/controllers/application_controller.rb:149]: I'm  wondering if we can refactor lines 149-152 out since we repeat several times below

**Review by emyl3 — CHANGES_REQUESTED** [review]
Great work, Josh! 🎉 Just noticed some things! Let me know if you have questions :D 
1. Visual/UX Review sheet - `line 7`
too much space between "Need help along the way" text compared to figma
**Dev**
Screen Shot 2020-10-01 at 10 10 40 AM**
**Figma**
Screen Shot 2020-10-01 at 10 10 55 AM

2. Visual/UX Review sheet - `line 10` 
still looks unresolved as the padding between a lot of the PE fields throughout seem off compared to figma:
**Dev**
Screen Shot 2020-10-01 at 10 22 29 AM
**Figma**
Screen Shot 2020-10-01 at 10 18 44 AM
3. Visual/UX Review sheet - `line 39` 
there is still a unnecessary comma if the user doesn't have an origin facility
Screen Shot 2020-10-01 at 10 32 05 AM

**Comment by JoshingYou1** [comment]
For issue #1, here's the margin in figma and the margin in the code:
Screen Shot 2020-10-01 at 11 12 40 AM
Screen Shot 2020-10-01 at 11 13 28 AM

For issue #2, I forgot to mention Becca recommended `16px` for all of the text above the labels. I posted a screenshot in Slack.

File: app/controllers/application_controller.rb (hunk 4 above)
Comment (JoshingYou1) [reply app/controllers/application_controller.rb]: Let me know what you think!

Comment (emyl3) [reply app/controllers/application_controller.rb]: that is what I was thinking! thank you josh!

**Review by emyl3 — APPROVED** [review]
Thanks for all the context, Josh! This looks good to me! :)

**Review by bradjohnson92008 — APPROVED** [review]
Fixes all look good Josh!

**Pull request merged**